README.md

```diff
@@ -66,10 +66,11 @@ After cloning don't forget to `git submodule update --init`.
 
 Before opening a PR make sure to:
 
-- Add tests that cover your changes.
-- `deno test --unstable --allow-all` passes.
-- `deno run -A _tools/format.js` passes without changing files.
-- `deno run -A _tools/lint.js` passes.
+- have the latest Deno version installed locally
+- add tests that cover your changes.
+- `deno task test:canary` passes.
```

Comment:
Should this just be `deno task test`? Looks like people won't necessarily be running canary (though, maybe it would be neat if it fetched the latest canary).

Reply:
It's somewhat required to run on canary